src/Widgets/ButtonWidget/ButtonWidgetComponent.tsx

```diff
@@ -20,11 +20,7 @@ provideComponent(ButtonWidget, ({ widget }) => {
 
   return (
     <WidgetTag className={alignmentClassNameWithBlock(widget.get('alignment'))}>
-      <LinkTag
-        to={target}
-        className={buttonClassNames.join(' ')}
-        style={{ minHeight: '40px', minWidth: '35px' }} // TODO: Check this by a designer
```

Comment:
A button without a label now more or less collapses: 
<img width="636" alt="Screenshot 2024-01-11 at 11 22 06" src="https://github.com/Scrivito/scrivito-portal-app/assets/86275/d1536708-7636-4824-a9a6-883e1ddfc172">

Please ensure, that it has at least a min height that is equal to the height with a text.

This is also important during loading - Scrivito will first render an "empty" button before rendering the button with the label, once the target obj is loaded.